File: haushaltsbuch/src/main/java/org/github/haschi/haushaltsbuch/api/_BeendeInventur.java
package org.github.haschi.haushaltsbuch.api;

import org.axonframework.commandhandling.TargetAggregateIdentifier;
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Aggregatkennung;
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Anweisung;
import org.immutables.value.Value;

@Anweisung
public interface _BeendeInventur
{
    @Value.Parameter
    @TargetAggregateIdentifier
    Aggregatkennung von();
}

File: haushaltsbuch/src/main/kotlin/org/github/haschi/haushaltsbuch/api/BeendeInventur.kt
package org.github.haschi.haushaltsbuch.api

import org.axonframework.commandhandling.TargetAggregateIdentifier
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Aggregatkennung

data class BeendeInventur(
        @TargetAggregateIdentifier val von: Aggregatkennung)
